src/math.cpp

```diff
@@ -33,15 +29,42 @@ namespace util {
 
     double normalize_angle(double angle)
     {
-        while (angle >= M_PI)
+        double a = angle;
+
+        if (angle > pi)
```

Comment:
Oh, @lightmare is there are reason you don't want to use `M_PI` from `global.hpp`?